app/frontend/actions.js

```diff
@@ -357,29 +359,53 @@ module.exports = ({setState, getState}) => {
   }
 
   const submitTransaction = async (state) => {
-    loadingAction(state, 'processing transaction', 'Submitting transaction...')
+    loadingAction(state, 'Submitting transaction...')
     try {
       const address = state.sendAddress.fieldValue
       const amount = state.sendAmount.coins
-      const sendResponse = await wallet.sendAda(address, amount)
-      const updatedBalance = await wallet.getBalance()
+      let sendResponse = await wallet.sendAda(address, amount)
       if (sendResponse.success) {
-        setTimeout(() => setState({sendResponse: ''}), 4000)
+        loadingAction(state, 'Transaction submitted - syncing wallet...')
```

Comment:
I would refactor this to the function something like `waitForBlockchain()` it will make code simpler and more easy to understand